static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,81 @@
+/* global SETTINGS: false */
+import React from 'react';
+import { connect } from 'react-redux';
+import Loader from 'react-loader';
+
+import Jumbotron from '../components/Jumbotron';
+import {
+  startProfileEdit,
+  FETCH_PROCESSING,
+  clearProfile,
+} from '../actions/index';
+import ProfileFormContainer from './ProfileFormContainer';
+
+class SettingsPage extends ProfileFormContainer {
+
+  componentDidMount() {
+    this.fetchProfile();
+  }
+
+  componentDidUpdate() {
+    this.fetchProfile();
+  }
+
+  componentWillUnmount() {
+    const { dispatch, params: { username } } = this.props;
```

Comment:
Yes, that's what I meant above, react-router injects the `params` prop because we declare the url parameter for the `/users/:username` page in the route declaration.

We don't need the username here at all, however, because on `/settings` we're never going to be editing another users profile.